### PR TITLE
add get_nameserver_config to Lumberg::Whm::Dns.

### PR DESCRIPTION
**Pull request by Tei1988**
I noticed that get_nameserver_config API is not implemented.
So, I implemented it.

Review please.

### CODE REVIEW

**Comment by rubemz** [comment]
@Tei1988 can you bump the version to `4.1.0`?

**Comment by rubemz** [comment]
@Tei1988 then we can merge and release the new version.
Thanks!

File: lib/lumberg/whm/dns.rb [hunk 1]
```diff
@@ -52,6 +52,15 @@ def get_zone_record(options = {})
         server.perform_request('getzonerecord', options)
       end
 
+
```
Comment (boone) [inline lib/lumberg/whm/dns.rb]: can you remove this extra newline?

**Comment by Tei1988** [comment]
Thank you all for review. I corrected them.

**Comment by rubemz** [comment]
@Tei1988 Thanks for this PR! Version `4.1.0` is out!

**Pull request merged**